src/js/construct-element.js

```diff
@@ -5,48 +5,48 @@ export default {
 	* Build a full alert message element. Used when there is no message element in the DOM.
 	* @returns {HTMLElement} Returns the new alert type message element
 	*/
-	alertMessage: (opts) => {
+	alertMessage: (el) => {
```

Comment:
Is this `el` a HTMLElement?

Reply:
no, its `this`, i just didn't really know what to call it. Any ideas?

Reply:
If you're happy getting rid of `messageClass` this can go back to passing an `opts` object?